UnityProject/Assets/Scripts/Lighting/LightSource.cs

```diff
@@ -107,7 +108,8 @@ public override void Trigger(bool newState)
 	public override void OnStartClient()
 	{
 		base.OnStartClient();
-		SyncLightState(mState, mState);
```

Comment:
This is required for late joiners. See previous comments about initializing SyncVars with hooks

Reply:
Agreed that would be necessary.